robocop/checkers/lengths.py

```diff
@@ -2,46 +2,50 @@
 from robocop.checkers import BaseChecker
 from robocop.messages import MessageSeverity
 
-MSGS = {
-    "0501": (
-        "too-long-keyword",
-        "Keyword is too long (%d/%d)",
-        MessageSeverity.WARNING
-    ),
-    "0502": (
-        "too-few-calls-in-keyword",
-        "Keyword have too few keywords inside (%d/%d)",
-        MessageSeverity.WARNING
-    ),
-    "0503": (
-        "too-many-calls-in-keyword",
-        "Keyword have too many keywords inside (%d/%d)",
-        MessageSeverity.WARNING
-    ),
-    "0504": (
-        "too-long-test-case",
-        "Test case is too long (%d/%d)",
-        MessageSeverity.WARNING
-    ),
-    "0505": (
-        "too-many-calls-in-test-case",
-        "Test case have too many keywords inside (%d/%d)",
-        MessageSeverity.WARNING
-    ),
-    "0506": (
-        "file-too-long",
-        "File has too many lines (%d/%d)",
-        MessageSeverity.WARNING
-    )
-}
-
 
 def register(linter):
     linter.register_checker(LengthChecker(linter))
 
 
 class LengthChecker(BaseChecker):
-    msgs = MSGS
+    msgs = {
+        "0501": (
+            "too-long-keyword",
+            "Keyword is too long (%d/%d)",
+            MessageSeverity.WARNING,
+            ('max_len', 'keyword_max_len', int)
+        ),
+        "0502": (
+            "too-few-calls-in-keyword",
+            "Keyword have too few keywords inside (%d/%d)",
+            MessageSeverity.WARNING,
+            ('min_calls', 'keyword_min_calls', int)
+        ),
+        "0503": (
+            "too-many-calls-in-keyword",
+            "Keyword have too many keywords inside (%d/%d)",
+            MessageSeverity.WARNING,
+            ('max_calls', 'keyword_max_calls', int)
+        ),
+        "0504": (
+            "too-long-test-case",
+            "Test case is too long (%d/%d)",
+            MessageSeverity.WARNING,
+            ('max_len', 'testcase_max_len', int)
+        ),
+        "0505": (
+            "too-many-calls-in-test-case",
+            "Test case have too many keywords inside (%d/%d)",
+            MessageSeverity.WARNING,
+            ('max_calls', 'testscase_max_calls', int)
+        ),
+        "0506": (
+            "file-too-long",
+            "File has too many lines (%d/%d)",
+            MessageSeverity.WARNING,
+            ('max_lines', 'file_max_lines', int)
```

Comment:
I will create documentation for every configurable, it;s related to #41 (we need to auto generate docs for it)